### ISSUE TITLE
Does it support yolo segmentation?

### ISSUE THREAD

**Issue by ghost** [kadirnar/torchyolo#37]
hi sir
yolo v5 v7 v8 provides segmentation function.
so does torchyolo support yolo segmentation?
i want to detect small object via segmentation.
thanks.

**Comment by kadirnar** [comment]
Yes I will add segmentation support. But I'm thinking of adding other yolo models first.(Yolop,Yolos,Yolov5-Lite) If you want to add, I can help.

ModelHub:
https://github.com/kadirnar/torchyolo/tree/main/torchyolo/modelhub

**Comment by ghost** [comment]
thanks,sir
Please notify me as soon as possible if it is completed.
your model is so great.

**Comment by kadirnar** [comment]
Thank you. I will add it soon.
